### ISSUE TITLE
Altitude labels are behind heat map 

### ISSUE THREAD

**Issue by mgunther68** [lignumaqua/dump1090#4]
I find that the altitude labels on aircraft seem to be underneath the heat map layer and sometimes hard to read. Can this be changed?

**Comment by lignumaqua** [comment]
Yes, I had noticed that as well. It's slightly tricky because of the way GMaps dynamically assigns z values. However, we should be able to put the labels on their own canvas over the top of everything. (The heat map is also a new canvas, which is why it's on top)

**Comment by lignumaqua** [comment]
Update. I realized that the default code never sets the zIndex of the aircraft icons. This means they render with the Google default - icons lower down the screen overlay those higher up. 

That's not what we want here, we want the zIndex of the icons (and labels) to follow the altitude of the aircraft, so higher aircraft overlay lower ones.

I've made that change (not pushed to github yet) and it seems to work well, and solve the heatmap issue.
